Dockerfile

```diff
@@ -1,8 +1,15 @@
 # Learn about building .NET container images:
 # https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md
+FROM node:latest AS node_base
```

Comment:
I think we should probably specify the node version, just to avoid any risk of breaking changes (although very unlikely!)
```suggestion
FROM node:18 AS node_base
```